opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -435,6 +460,7 @@ namespace Opm {
                 return well;
             }
         }
+        //this is not thread safe...
```

Comment:
This comment, in isolation, is wrong. If the calling scope ensures catching and communication all is fine.